cloudbuild-prod.yaml

```
@@ -20,6 +20,7 @@ steps:
   - name: 'gcr.io/cloud-builders/docker'
     args:
       - 'build'
+      - '--platform=linux/arm64'
```

this will then not build for `amd64`, right?

Should we not use the multi-arch docker build? Like this

```suggestion
      - '--platform=$TARGETPLATFORM'
```